Scripts/Core/EditorVR.DirectSelection.cs

```diff
@@ -37,9 +37,9 @@ internal float GetPointerLength(Transform rayOrigin)
 					{
 						var miniWorld = ray.miniWorld;
 
-						// As the miniworld gets smaller, the ray length grows, hence localScale.Inverse().
+						// As the miniworld gets smaller, the ray length grows, hence lossyScale.Inverse().
 						// Assume that both transforms have uniform scale, so we just need .x
-						length *= miniWorld.referenceTransform.TransformVector(miniWorld.miniWorldTransform.localScale.Inverse()).x;
+						length *= miniWorld.referenceTransform.TransformVector(miniWorld.miniWorldTransform.lossyScale.Inverse()).x;
```

Comment:
Yep. There are a few more places where I needed to replace localScale with lossyScale, in case you want to check those out ahead of time.